### ISSUE TITLE
Don't require creating an account or password

### ISSUE THREAD

**Issue by BryanQuigley** [davidpanderson/science_united#2]
To help increase usage of boinc let's remove the account requirement.  Allow someone to download the client and contribute without create an account, giving their email, or a password.    Each one of those items, adds steps some people won't do.

It may need machine accounts on the backend, but incentives to cheat, etc should really go away.

**Comment by davidpanderson** [comment]
I agree.  It wouldn't be hard to implement.  I think there should be some simple mechanism for such a user to connect to their account later on, and assign an email address and password to it.  This would require a small change to the client and Manager.

**Comment by BryanQuigley** [comment]
Seems like it would require a redesign at this point - guessing having an email is useful for filtering, etc.  Closing old issues.
